### PR TITLE
Exclude kauailabs from linkcheck

### PR DESCRIPTION
**Pull request by Daltz333**
KauaiLabs website consistently has downtime, and various website issues. This excludes all urls with their root domain from the link check. In the mean time, I have sent them an email informing them of these issues.

### CODE REVIEW

**Comment by AustinShalit** [comment]
Is this good now?

**Comment by Daltz333** [comment]
Yep.

File: source/conf.py [hunk 1]
```diff
@@ -47,6 +47,9 @@
 # Disable following anchors in URLS for linkcheck
 linkcheck_anchors = False
 
+# Linkcheck Exclusions
+linkcheck_ignore = [r'.*kauailabs.com.*']
```
Comment (AustinShalit) [inline source/conf.py]: This will ignore anything with `kauailabs.com` in the address. Is that what we want to do or do we want to be more selective?

Comment (sciencewhiz) [reply source/conf.py]: That's probably more open then it needs to be, but I think it's also not going to cause a problem. It's highly unlikely that someone else makes a domain name that contains kauailabs.com that we would reference in the docs.

**Pull request merged**